src/Sylius/Bundle/UserBundle/Form/Type/CustomerSimpleRegistrationType.php

```diff
@@ -51,6 +51,8 @@ public function buildForm(FormBuilderInterface $builder, array $options = [])
             ])
             ->add('user', 'sylius_user_registration', [
                 'label' => false,
+                'validation_groups' => $this->validationGroups,
```

Comment:
I have problems with this i don't know why but embedded form type has only `sylius` and `default` validation group and without this change validation doesn't work.